docstring_parser/parser/google.py

```diff
@@ -0,0 +1,135 @@
+"""Google-style docstring parsing."""
+
+import inspect
+import re
+
+from .common import ParseError, Docstring, DocstringMeta
+
+_sections = {
+    'Arguments': 'param',
+    'Args': 'param',
+    'Parameters': 'param',
+    'Params': 'param',
+    'Raises': 'raises',
+    'Exceptions': 'raises',
+    'Except': 'raises',
+    'Attributes': None,
+    'Example': None,
+    'Examples': None,
+    'Returns': 'returns',
+    'Yields': 'returns',
+}
+_titles = '|'.join('(%s)' % t for t in _sections)
+_valid = {t for t, a in _sections.items() if a}
+
+
+def _build_meta(text: str, title: str) -> DocstringMeta:
+    """Build docstring element.
+
+    :param text: docstring element text
+    :param title: title of section containing element
+    :return:
+    """
+
+    meta = _sections[title]
+    if meta == 'returns' and ':' not in text.split()[0]:
+        return DocstringMeta([meta], description=text)
+
+    # Split spec and description
+    before, desc = text.split(':', 1)
+    if desc:
+        desc = desc[1:] if desc[0] == ' ' else desc
+        if '\n' in desc:
+            first_line, rest = desc.split('\n', 1)
+            desc = first_line + '\n' + inspect.cleandoc(rest)
+        desc = desc.strip('\n')
+
+    # Build Meta args
+    m = re.match(r'(\S+) \((\S+)\)$', before)
+    if meta == 'param' and m:
+        arg_name, type_name = m.group(1, 2)
+        args = [meta, type_name, arg_name]
+    else:
+        args = [meta, before]
```

Comment:
This can cause problems for `DocstringMeta` if `meta` is `None`, and it may be the case in case you hit `None` mapping such as `Example`, `Examples` or `Attributes`. This should be either dealt with here, or there should be added support for the missing mappings.

Reply:
There already is a mechanism to completely skip handling of of those values:
```python
chunks = {}
for j, (start, end) in enumerate(splits):
    title = matches[j].group(1)
    if title not in _valid:
        continue
    chunks[title] = meta_chunk[start:end].strip('\n')
```

Reply:
Gotcha